src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -521,6 +708,90 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
 }];
 ```
 
+<Variant platform="ios" api="update-propositions-withCallback-withTimeout" repeat="12"/>
```

Comment:
Add reference to this in `/adobe-journey-optimizer-decisioning/api-reference.md`

Reply:
done @spoorthipujariadobe